src/twisted/application/twist/test/test_options.py

```diff
@@ -47,7 +47,7 @@ def patchOpen(self) -> None:
         """
         self.opened = []  # type: List[Tuple[str, Optional[str]]]
 
-        def fakeOpen(name: str, mode: Optional[str] = None) -> NotImplemented:
+        def fakeOpen(name: str, mode: Optional[str] = None) -> Exception:
```

Comment:
```suggestion
        def fakeOpen(name: str, mode: Optional[str] = None) -> object:
```

https://github.com/python/mypy/issues/363